data-migrations/2025-11-19-1427-init-stored-procs.sql

```diff
@@ -70,12 +65,11 @@ END;
 
 START TRANSACTION;
 
--- works: insert new works (DOIs) and skip rows that would violate the doi unique constraint
 INSERT IGNORE INTO works (doi)
 SELECT doi
 FROM stagingWorkVersions;
```

Comment:
We should change `utf8mb4_unicode_ci` to `utf8mb4_0900_ai_ci` in this file. I did a search to double check if `utf8mb4_unicode_ci` is anywhere else and they aren't in any other files anymore.